src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -372,6 +449,15 @@ class StorageTab extends React.Component {
           choices: this.props.storageClasses.map(getName),
           initialValue: '--- Select ---',
         },
+        undefined,
```

Comment:
Please introduce constant for that, like `FIELD_READ_ONLY` or similar

Reply:
Follow up https://github.com/kubevirt/web-ui-components/issues/133